packages/server-core/src/media/static-resource-filters/static-resource-filters.hooks.ts

```diff
@@ -38,7 +39,7 @@ export default {
 
   before: {
     all: [
-      verifyScope('admin', 'admin'),
+      iff(isProvider('external'), verifyScope('static_resource', 'read')),
```

Comment:
It seems like we should disallow all methods other than `get`, since that appears to be the only method we use. We should then move this verifyScope to `get`, since in case we ever decide to implement create/update/patch/delete, they would likely require `static_resource:write`.